examples/datasets/plot_limo_data.py

```diff
@@ -111,68 +169,250 @@
+                                  -limo_epochs['Face/B'].average()],
+                                 weights='equal')
+
+# plot difference wave
 difference_wave.plot_joint(times=[.15], title='Difference Face A - Face B')
 
 ###############################################################################
-# As expected, no see clear differential patterns appears when contrasting
-# Face A and Face B. However, we could narrow our search to
-# since this is a "visual paradigm" it might be best to electrodes located over
-# the occipital lobe. After all this is "visual paradigm". Thus, differences
-# between stimuli (if any) might easier to spot over "more visual areas".
-#
+# As expected, no see clear differential pattern appears when contrasting
+# Face A and Face B. However, we could narrow our search a little bit more.
+# Since this is a "visual paradigm" it might be best to look at electrodes
+# located over the occipital lobe, as differences between stimuli (if any)
+# might easier to spot over visual areas.
+
 # Create a dictionary containing the evoked responses
 conditions = ["Face/A", "Face/B"]
 evoked_dict = dict()
 for condition in conditions:
     evoked_dict[condition] = limo_epochs[condition].average()
-print(evoked_dict)
 
-# concentrate on an occipital electrode
+# concentrate analysis an occipital electrodes (e.g. B11)
 pick = evoked_dict["Face/A"].ch_names.index('B11')
 
 # compare evoked responses
-mne.viz.plot_compare_evokeds(evoked_dict, picks=pick)
+plot_compare_evokeds(evoked_dict, picks=pick, ylim=dict(eeg=(-15, 5)))
+
+###############################################################################
+# As expected, the difference between Face A and Face B are very small.
+
+###############################################################################
+# Visualize effect of stimulus phase-coherence
+# --------------------------------------------
+#
+# For visualization purposes, we can transform the phase-coherence
+# variable back to fit roughly to it's original scale. Phase-coherence
+# determined whether a face stimulus could be identified as such. Thus,
+# one could expect that faces with high phase-coherence should evoke stronger
+# activation patterns along occipital electrodes.
+#
+# As phase-coherence variable is a continuous variable, we'll need to
+# split the values of face coherence to percentiles to match roughly their
+# original scale (i.e, 0 to 100 % signal-to-noise ratio in 10% steps).
+# **Note:** We'll only do this for visualization purposes,
+
+name = "phase-coherence"
+factor = 'factor_' + name
+
+# create phase-coherence percentiles
+df = limo_epochs.metadata
+df[factor] = pd.cut(df[name], 11, labels=False) / 10
+# overwrite metadata
+limo_epochs.metadata = df
+
+# color scheme for percentile plot
+colors = {str(val): val for val in np.sort(df[factor].unique())}
+# compute evoked for each phase-coherence percentile
+evokeds = {str(val): limo_epochs[factor + ' == ' str(val)].average()
```

Comment:
SyntaxError

https://circleci.com/gh/mne-tools/mne-python/16326

Reply:
Yep, I can make the changes tonight (sorry, had to leave this „undone“ earlier this morning)